src/GeneratedHydrator/Configuration.php

```diff
@@ -220,4 +225,20 @@ public function getClassNameInflector()
 
         return $this->classNameInflector;
     }
+
+    /**
+     * @param \PhpParser\NodeVisitor[] $visitors
+     */
+    public function setCustomVisitors(array $customVisitors)
```

Comment:
The configuration object is a pile of mud on purpose: it serves as
mini-factory for the entire lib.
On Sep 26, 2015 01:27, "Jefersson Nathan" <notifications@github.com> wrote:

> In src/GeneratedHydrator/Configuration.php
> <https://github.com/Ocramius/GeneratedHydrator/pull/40#discussion_r40477394>
> :
>
> > @@ -220,4 +225,20 @@ public function getClassNameInflector()
> >
> >          return $this->classNameInflector;
> >      }
> > +
> > +    /**
> > +     * @param \PhpParser\NodeVisitor[] $visitors
> > +     */
> > +    public function setCustomVisitors(array $customVisitors)
>
> What about __construct @Ocramius <https://github.com/Ocramius> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Ocramius/GeneratedHydrator/pull/40/files#r40477394>.
>